Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Use the official ROS 2 humble base image
+FROM ros:humble
+
+# Set the working directory
+WORKDIR /root/ros2_ws/src
+
+COPY . .
+# Install additional dependencies
+RUN apt-get update \
+    && apt-get -y --quiet --no-install-recommends install \
+    gcc \
+    git \
+    libxml2-dev \
+    libxslt-dev \
+    python3 \
+    ros-humble-ament-cmake-clang-format\
+    ros-humble-rviz2
```

Comment:
https://www.ros.org/reps/rep-2001.html#:~:text=ros_base%20%3A%20The%20ros_base%20variant%20composes,not%20contain%20any%20GUI%20dependencies.

This image installs the desktop version, which includes rviz2.

Reply:
I got it but If I try it without install rviz2. I got the following error;

![Screenshot from 2024-01-19 15-39-43](https://github.com/ethz-asl/grid_map_geo/assets/39880147/ddaefa0f-0c56-4e83-a986-4ac9403fc958)

Reply:
Ah, add this to the `package.xml` : `<exec_depend>rviz2</exec_depend>`

Reply:
https://github.com/avcuenes/grid_map_geo/pull/1